### PR TITLE
attempt at issue #66

### PR DESCRIPTION
**Pull request by galegosimpatico**

### CODE REVIEW

File: app/assets/javascripts/comments.js [hunk 1]
```diff
@@ -32,9 +36,17 @@ $(() => {
     else {
       QPixel.createNotification('danger', data.message);
     }
+
+    // On success, the `Post` button, which has been re-labeled as `Posted`, is not yet re-labeled `Post` when reaching
+    // this line. Gets re-labeled again by the line below.
+    $(evt.target).find('.form-group-horizontal > .actions > input').attr('value', 'Post')
```
Comment (ArtOfCode-) [inline app/assets/javascripts/comments.js:42]: Easier selector: `$(evt.target).find('input[type="submit"]')`

**Review by ArtOfCode- — CHANGES_REQUESTED** [review]

**Comment by ArtOfCode-** [comment]
Has this been tested locally?

**Comment by galegosimpatico** [comment]
Before and after [your review][0], using `debugger` breaks, Opera on macOS.

"Works for me", but no auto tests, no selenium, no chromium, no Firefox... who knows? Hopefully it does what is expected. 🤷🏿

[0]: https://github.com/codidact/qpixel/pull/155#pullrequestreview-477348985

**Comment by ArtOfCode-** [comment]
Simple enough that some basic manual testing is enough for me.

**Pull request merged**